pkg/uroot/builder/bb/bb.go

```diff
@@ -100,10 +100,16 @@ func BuildBusybox(env golang.Environ, pkgs []string, binaryPath string) error {
 	var bbPackages []string
 	// Move and rewrite package files.
 	importer := importer.For("source", nil)
+	seenPackages := map[string]bool{}
 	for _, pkg := range pkgs {
-		if _, ok := skip[path.Base(pkg)]; ok {
+		basePkg := path.Base(pkg)
+		if _, ok := skip[basePkg]; ok {
 			continue
 		}
+		if _, ok := seenPackages[path.Base(pkg)]; ok {
+			return fmt.Errorf("Failed to build busybox: found duplicate pkgs %s", basePkg)
```

Comment:
we say "-build=bb", the only question I have here is that we should just say "Failed to build:"? 
I worry the busybox guys will feel like we are taking their name or something.

But really this is fine.

Reply:
I think you can say 
#fixes 1178
or something in the commit and it will close that issue.

Reply:
Sounds good! I'll change the error :)